windows/tools/tag_scanner_window.py

```diff
@@ -480,5 +477,5 @@ def tag_specific_scan(self, tag, errors):
         if err:
             errors[cls] = "%s\n%s:\n%s\n" % (
                 errors.get(cls, ""),
-                tag.filepath.split(self.tags_dir, 1)[-1],
+                str(tag.filepath).split(str(self.handler.tagsdir), 1)[-1],

```

Comment:
What actually is this code?

Reply:
Piecing together a tag-specific error string. I have cleaned it up to make it more clear.
```python
rel_tag_path = str(tag.filepath.relative_to(self.handler.tagsdir))
errors[cls] = (
    errors.get(cls, "") + "\n" +
    rel_tag_path + "\n" +
    err + "\n"
    )
```